Cargo.toml

```diff
@@ -1,14 +1,13 @@
 [package]
 name = "app-boilerplate-rust"
-version = "1.0.0"
+version = "1.1.0"
 authors = ["yhql", "agrojean-ledger"]
 edition = "2021"
 
 [dependencies]
-ledger_device_sdk = "1.0.1"
-ledger_device_ui_sdk = "1.1.1"
+ledger_device_sdk = {git = "https://github.com/LedgerHQ/ledger-device-rust-sdk.git", version = "1.1.0"}
```

Comment:
Not sure we want to introduce this change in the context of this app. Why would we want to allow two different versions to be used ? We could have breaking changes on the main branch on the git repo that wouldn't be available in the crates.io version, couldn't we ?

Reply:
Not two different versions, as the version field is used even if we retrieve the crates from `git`